man/systemd-system.conf.xml

```diff
@@ -525,6 +525,18 @@
         details. Note that this setting has no effect on the OOM score adjustment value of the service
         manager process itself, it retains the original value set during its invocation.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>DefaultSmackProcessLabel=</varname></term>
+
+        <listitem><para>Takes a <option>SMACK64</option> security label as the argument. The process executed
+        by a unit will be started under this label if <varname>SmackProcessLabel=</varname> is not set in the
+        unit. See <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for the details.</para>
+
+        <para>If the value is <literal>/</literal> only labels labels specified with <varname>SmackProcessLabel=</varname>
```

Comment:
'-' is actually used in a special way (see `-CIPSO` and `-DELETE`) and I'd like to stay away from it, even if only to avoid confusion (LSM/MAC stuff is complicated enough) between the ways the kernel and systemd use '-'.

Reply:
hmm, can we use `~`?

Reply:
I am afraid we can't

> Single character labels using special characters, that being anything other than a letter or digit, are reserved for use by the Smack development team.